service/src/main/java/eu/europa/ec/fisheries/ers/service/mapper/FACatchSummaryMapper.java

```diff
@@ -296,16 +296,16 @@ protected SummaryTableDTO getSummaryTableDTOWithPresentation(List<FaCatchSummary
 
     @NotNull
     private Map<FishSizeClassEnum, Object> populateFishSizeClassMapForOnlyForFishClass(FaCatchSummaryCustomProxy customEntity, Double speciesCnt, Map<FishSizeClassEnum, Object> fishSizeSummaryMap) {
```

Comment:
Object as value type is not very nice, especially since it appears to be Double

Reply:
You are very right, but the map comes from a DTO class that I don't want to touch